src/components/VideoCart.tsx

```diff
@@ -17,12 +18,20 @@ export type videoCartProps = {
     videoId?: number;
     timestamp: number | '';
     includeAvatar: boolean;
+    includeStreamerName: boolean;
     title: string;
     imageId: number;
 }
 const VideoCart = (props: videoCartProps) => {
-    let { id, streamer, player, streamStart, action, videoId, imageId = '', title = '', timestamp = '', includeAvatar = false } = props
+    let { id, streamer, player, streamStart, action, videoId, imageId = '', title = '', timestamp = '', includeAvatar = false, includeStreamerName } = props
     videoId = videoId || id
+    var newTitle
+    if (includeStreamerName && (typeof streamer != 'undefined')) {

```

Comment:
it should be includeStreamerName && typeof streamer !== 'undefined'
or u can do includeStreamerName && streamer != null